extensions/grpc/codegen/src/main/java/io/quarkus/grpc/deployment/GrpcCodeGen.java

```diff
@@ -94,6 +95,14 @@ public boolean trigger(CodeGenContext context) throws CodeGenException {
         return false;
     }
 
+    private String escapeWhitespace(String path) {
+        if (OS.determineOS() == OS.LINUX) {
+            return path.replaceAll(" ", "\\ ");
+        } else {
+            return path;
```

Comment:
spaces will work as is on Windows and macOS?

Reply:
I don't know, I don't really have a way to check.
@rsvoboda reported that it doesn't work on linux

Reply:
To shed some more light on it, the paths are passed as separate arguments in the `ProcessBuilder`'s command, so they should be okay without backslashes before spaces. It's `protoc` that doesn't handle them properly on Linux. I don't know about other systems.

Reply:
Maybe @cescoffier can check on macOS if you can put together a test case?

Reply:
@cescoffier checked and this version (without `\` before spaces) worked for him on Mac